### PR TITLE
Cleaning up plugin

### PR DESCRIPTION
**Pull request by sheivin**

### CODE REVIEW

File: src/main/java/co/cask/PluginJSON.java [hunk 1]
```diff
@@ -351,18 +347,29 @@ private void printHeader() {
...
   private void printFooter() {
-    getLog().info(StringUtils.repeat('-', 72));
+    getLog().info(repeat("-", 72));
+  }
+
+  /**
+   * Repeats text for specified number of times
+   */
+  private String repeat(String text, int numRepetitions) {
+    String repeated = "";
```
Comment (albertshau) [inline src/main/java/co/cask/PluginJSON.java]: it's better to use a StringBuilder here instead of repeated concats

**Review by albertshau — COMMENTED** [review]
i think we can remove the profiles section from the pom too

File: src/main/java/co/cask/PluginJSON.java (hunk 1 above)
Comment (sheivin) [reply src/main/java/co/cask/PluginJSON.java]: Yeah, that is a better choice. I will look into the profiles.

**Comment by sheivin** [comment]
@albertshau you were right. Looks like the profiles section was not needed.

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**